Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -46,7 +51,10 @@ export function documentOwnershipByIdSql(documentId: string): string {
  * @returns SQL query string to retrieve references
  */
 export function checkForReferencesByDocumentId(documentId: string[]): string {
```

Comment:
This function name: another reason to consider renaming `existence` to `references`.